os/board/rtl8721csm/src/rtl8721csm_i2schar.c

```diff
@@ -0,0 +1,132 @@
+ *    distribution.
+ * 3. Neither the name NuttX nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+#include <sys/types.h>
+#include <errno.h>
+#include <debug.h>
+#include <tinyara/clock.h>
+
+#if defined(CONFIG_AUDIO_I2SCHAR) && defined(CONFIG_AMEBAD_I2S)
+
+#include <tinyara/audio/i2s.h>
+#include "amebad_i2s.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#ifndef CONFIG_AMEBAD_I2SCHAR_MINOR
+#define CONFIG_AMEBAD_I2SCHAR_MINOR 0
+#endif
+/*****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+static void err_cb(FAR struct i2s_dev_s *dev, FAR void *arg, int flags)
+{
+/* Do something here */
+}
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: i2schar_devinit
+ *
+ * Description:
+ *   All architectures must provide the following interface in order to
+ *   work with apps/examples/i2schar.
+ *
+ ****************************************************************************/
+
+int i2schar_devinit(void)
+{
+	static bool initialized;
+	struct i2s_dev_s *i2s;
+	int ret;
+
+	/* Have we already initialized? */
+
+	if (!initialized) {
+		/* Call amebad_i2s_initialize() to get an instance of the I2S interface */
+
+		i2s = amebad_i2s_initialize(0);
+		if (!i2s) {
+			auddbg("ERROR: Failed to get the AMEBAD I2S driver\n");
```

Comment:
`auddbg` is for audio debug msg.
Please use proper debug function.

Reply:
Modified 'auddbg' to 'lldbg'.